widgetssdk/src/main/java/com/glia/widgets/chat/adapter/UploadAttachmentAdapter.kt

```diff
@@ -271,9 +274,9 @@ class ViewHolder(
         inDeterminate: Boolean = true
     ) {
         progressIndicator.apply {
+            isIndeterminate = inDeterminate
```

Comment:
Does only changing the order of operations fix the issue?

Reply:
Yes, with the old order, the view didn't get `invalidated`, and as a result, the progress which was set before changing mode to `indeterminate=false` didn’t get applied to the view.